torchaudio/datasets/dr_vctk.py

```diff
@@ -0,0 +1,116 @@
+from pathlib import Path
+from typing import Dict, Tuple, Union
+
+from torch import Tensor
+from torch.utils.data import Dataset
+
+import torchaudio
+from torchaudio.datasets.utils import (
+    download_url,
+    extract_archive,
+    validate_file,
+)
+
+
+_URL = "https://datashare.ed.ac.uk/bitstream/handle/10283/3038/DR-VCTK.zip"
+_CHECKSUM = "29e93debeb0e779986542229a81ff29b"
+_SUPPORTED_SUBSETS = {"train", "test"}
+
+
+class DR_VCTK(Dataset):
+    """Create a dataset for Device Recorded VCTK (Small subset version).
+
+    Args:
+        root (str or Path): Root directory where the dataset's top level directory is found.
+        download (bool):
+            Whether to download the dataset if it is not found at root path. (default: ``False``).
+        url (str): The URL to download the dataset from.
+            (default: ``"https://datashare.ed.ac.uk/bitstream/handle/10283/3038/DR-VCTK.zip"``)
+        subset (str): The subset to use. Can be one of ``"train"`` and ``"test"``. (default: ``"train"``).
+    """
+
+    def __init__(
```

Comment:
Can we reorder the signature so that more used ones come earlier, and we can still add positional arguments?

```python
def __init__(self, root, subset, *, download, url):
```

Reply:
The asterisk to make `download` and `url` keyword-only argument is missing.